python/dendro/iaac/aws_batch_job_definition/job_definition.py

```diff
@@ -64,16 +63,22 @@ def create_job_definition(
 
     mount_points = [
         {
-            'containerPath': 'string',
+            'containerPath': '/mount/efs',
             'readOnly': False,
             'sourceVolume': f'{job_definition_name}-EfsVolume'
         },
     ]
 
+    # Container Command
+    container_command = ["python", "main.py"]
```

Comment:
@luiztauffer Should this be "/app/main.py" instead of just "main.py"?

Reply:
@magland this is just a placeholder for now, but sure, we should use the default command for dendro apps
fyi, this is something we can override in two occasions:
- when creating the Job Definition, using the `container_command_override` arg of this function
- when submitting jobs